components/ProjectList.js

```diff
@@ -66,56 +75,71 @@ const Wrapper = styled.div`
   align-items: center;
 `;
 
-const SearchInput = styled.input`
+const SearchWrapper = styled.div`
   position: fixed;
-  top: 5em;
+  top: 10em;
   width: 85%;
   margin: 5px;
-  padding: 10px;
+  padding: 4px;
   border-radius: 7px;
-  border: 1px solid grey;
+  border: 2.5px solid #94c3dd;
+  box-shadow: 0 4px 14px 0 rgb(0 0 0 / 18%);
+`;
+
+const StyledSearchIcon = styled(SearchIcon)`
+  position: absolute;
+  top: 0.4em;
+`;
+const SearchInput = styled.input`
+  width: 95%;
+  margin-left: 1em;
   font-size: 15px;
   word-wrap: break-word;
   word-break: break-all;
-  border: none;
   font-size: 14px;
   color: #696969;
-  box-shadow: 0 4px 14px 0 rgb(0 0 0 / 10%);
+  border: none;
+  &:focus {
+    border-color: none;
+    outline: none;
+  }
 `;
 
 const ProjectWrapper = styled.div`
   position: fixed;
```

Comment:
This is just an idea for later not a remark that you should consider now: It may be cool to think about working with grid for overall app layout. I think if you work with a grid layout you could reduce much of your stylings that you solved with positioning.